packages/common/DynamoDb.js

```diff
@@ -26,33 +18,18 @@ const { RecordDoesNotExist } = require('./errors');
  * @static
  * @kind function
  */
-const get = aws.improveStackTrace(
-  async ({
+const get = async ({
```

Comment:
This could be simplified to:
```js
const get = async (params) => {
  deprecate('@cumulus/common/DynamoDb.get', '1.17.1', '@cumulus/aws-client/DynamoDb.get');
  return DynamoDb.get(params);
};
```